Loaded dependency [python3/3.10.13]: gcc/11.4.0-binutils-2.40
Loaded dependency [python3/3.10.13]: sqlite3/3.44.2
Loaded module: python3/3.10.13

Loading python3/3.10.13
  Loading requirement: gcc/11.4.0-binutils-2.40 sqlite3/3.44.2
Traceback (most recent call last):
  File "/appl/python/3.10.13/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/appl/python/3.10.13/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/zhome/de/d/169059/Desktop/irl-chess/project/sunfish_permuted.py", line 64, in <module>
    df = pd.read_csv(datapaths[0], index_col=None)
IndexError: list index out of range
